audeer/core/utils.py

```diff
@@ -309,6 +309,13 @@ def git_repo_version(
 def is_semantic_version(version: str) -> bool:
     r"""Check if given string represents a `semantic version`_.
 
+    Your version is allowed to start with a ``v``.
+    Otherwise it has to comply to ``X.Y.Z``,
```

Comment:
Here "otherwise" sounds like this only applies if it does not start with `v`. Maybe better "Apart from that"?

Reply:
I incorporated 'v' now in the second sentence:

![image](https://user-images.githubusercontent.com/173624/107257383-e3f40680-6a3a-11eb-9db3-2616942c84fb.png)